python/pyarrow/tests/test_csv.py

```diff
@@ -319,6 +321,79 @@ def test_header_skip_rows(self):
             "kl": ["op"],
         }
 
+    def test_skip_rows_after_names(self):
```

Comment:
Can you perhaps add a test with both `skip_rows` and `skip_rows_after_names`?

Reply:
Done. Would you like more is just the one good?

Reply:
It's good, thank you!